### ISSUE TITLE
Permission denied when saving model

### ISSUE THREAD

**Issue by arditobryan** [MinishLab/model2vec#34]
```
from model2vec.distill import distill

# Choose a Sentence Transformer model
model_name = "sentence-transformers/all-MiniLM-L6-v2"

# Distill the model
m2v_model = distill(model_name=model_name, pca_dims=256)

# Save the model
m2v_model.save_pretrained("m2v_model")

ERROR

100%|██████████| 30/30 [00:44<00:00,  1.50s/it]
---------------------------------------------------------------------------
PermissionError                           Traceback (most recent call last)
Cell In[186], [line 7](vscode-notebook-cell:?execution_count=186&line=7)
      [4](vscode-notebook-cell:?execution_count=186&line=4) model_name = "sentence-transformers/all-MiniLM-L6-v2"
      [6](vscode-notebook-cell:?execution_count=186&line=6) # Distill the model
----> [7](vscode-notebook-cell:?execution_count=186&line=7) m2v_model = distill(model_name=model_name, pca_dims=256)
      [9](vscode-notebook-cell:?execution_count=186&line=9) # Save the model
     [10](vscode-notebook-cell:?execution_count=186&line=10) m2v_model.save_pretrained("m2v_model")

File c:\Users\user1\miniconda3\envs\env1\Lib\site-packages\model2vec\distill\__main__.py:80, in distill(model_name, vocabulary, device, pca_dims, apply_zipf)
     [78](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/__main__.py:78) vocab = tokenizer.get_vocab()
     [79](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/__main__.py:79) wrong_token_ids = [vocab[token] for token in wrong_tokens]
---> [80](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/__main__.py:80) tokenizer = remove_tokens(tokenizer, wrong_tokens)
     [81](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/__main__.py:81) embeddings = np.delete(embeddings, wrong_token_ids, axis=0)
     [82](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/__main__.py:82) logger.info("Removed unused tokens from the tokenizer and embeddings.")

File c:\Users\user1\miniconda3\envs\env1\Lib\site-packages\model2vec\distill\tokenizer.py:25, in remove_tokens(tokenizer, tokens_to_remove)
     [23](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/tokenizer.py:23) with NamedTemporaryFile(mode="w+") as temp_file:
     [24](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/tokenizer.py:24)     tokenizer.save(temp_file.name)
---> [25](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/tokenizer.py:25)     data = json.load(open(temp_file.name))
     [26](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/tokenizer.py:26)     vocab: dict[str, int] = data["model"]["vocab"]
     [28](file:///C:/Users/user1/miniconda3/envs/env1/Lib/site-packages/model2vec/distill/tokenizer.py:28)     n_tokens = len(vocab)

PermissionError: [Errno 13] Permission denied: 'C:\\Users\\user1\\AppData\\Local\\Temp\\tmp8dwbcz_7'
```

**Comment by stephantul** [comment]
Hey @arditobryan ,

This is a bug and is already fixed in #30 , we'll do a release ASAP and will let you know. 

Thanks!

**Comment by stephantul** [comment]
As a temporary work-around, you can just clone this repository, and install it using `pip install /path/to/where/you/cloned` while in the venv where you wanted to store this.

**Comment by tomaarsen** [comment]
Or you can use:
```
pip install git+https://github.com/MinishLab/model2vec.git
```
to install the `main` branch directly into your git.

- Tom Aarsen

**Comment by stephantul** [comment]
Hello @arditobryan ,

This is solved in the current pypi release, 0.2.0. I'm closing this for now, let me know if you run into any further issues.

Stéphan
